plugins/s3_auth/Makefile.inc

```diff
@@ -16,3 +16,12 @@
 
 pkglib_LTLIBRARIES += s3_auth/s3_auth.la
 s3_auth_s3_auth_la_SOURCES = s3_auth/s3_auth.cc
+
+#GOOGLETEST_DIR=/opt/gtest
```

Comment:
@SolidWallOfCode this was not meant to run, since we don't have the plumbing necessary or even no agreement if we should use GTest (this was just temporary hack to be able to write the test itself in my env). 

This is WIP and pending the decision on the email thread "Unit Test Framework" on dev@trafficserver, might need to be converted to "make check" or anything else we decide to use.